steps/delius/event/create-event.ts

```diff
@@ -38,7 +38,7 @@ export async function createEvent(page: Page, { crn, allocation, event, date }:
     await fillDate(page, '#OffenceDate', _date)
     await fillDate(page, '#ConvictionDate', _date)
     await selectOption(page, '#MainOffence', null, option => !option.startsWith('('))
-    createdEvent.court = await selectOption(page, '#Court')
+    createdEvent.court = await selectOption(page, '#Court[name="Court"]')
```

Comment:
Do you need this extra name selector?  Selecting by ID is as specific as you can get

Reply:
the test has been failing on this id alone randomly. when i checked the page, I can see many selectors with id as 'Court'  at that point.